project.clj

```diff
@@ -56,9 +56,9 @@
                  [slingshot "0.12.2"]
 
                  ;; Database connectivity
-                 [com.zaxxer/HikariCP "2.4.3"]
+                 [com.zaxxer/HikariCP "2.4.5"]
```

Comment:
@ajroetker Your travis failures are caused by https://github.com/brettwooldridge/HikariCP/issues/600. We either need to use the old version, or hack up 2.4.5 to avoid the issue.